packages/cli/src/commands/git/index.ts

```diff
@@ -0,0 +1,94 @@
+import chalk from 'chalk';
+import Client from '../../util/client';
+import { ensureLink } from '../../util/ensure-link';
+import getArgs from '../../util/get-args';
+import getInvalidSubcommand from '../../util/get-invalid-subcommand';
+import handleError from '../../util/handle-error';
+import logo from '../../util/output/logo';
+import { getPkgName } from '../../util/pkg-name';
+import validatePaths from '../../util/validate-paths';
+import connect from './connect';
+import disconnect from './disconnect';
+
+const help = () => {
+  console.log(`
+  ${chalk.bold(`${logo} ${getPkgName()} git`)} <command>
+
+  ${chalk.dim('Commands:')}
+
+    connect                          Connect a Git provider repository to your project
```

Comment:
```suggestion
    connect                          Connect your git config "origin" remote as a Git provider to your project
```

Can we have this say where the remote repo comes from?